tests/main.test.ts

```diff
@@ -1021,76 +1021,93 @@ describe(`handle integrity checks`, () => {
       });
     });
   });
-  it(`should return an error when signature does not match with a tag`, async () => {
+  it(`fails on invalid signature for version range`, async () => {
     process.env.TEST_INTEGRITY = `invalid_signature`; // See `_registryServer.mjs`
 
     await xfs.mktempPromise(async cwd => {
       await expect(runCli(cwd, [`pnpm@1.x`, `--version`], true)).resolves.toMatchObject({
         exitCode: 1,
-        stderr: /Signature does not match/,
+        stderr: expect.stringMatching(/Error: Signature does not match/),
         stdout: ``,
       });
+    });
+  });
+
+  // FIXME: the test registry does not implement tag fetching yet
+  it.fails(`fails on invalid signature for tag`, async () => {
+    process.env.TEST_INTEGRITY = `invalid_signature`; // See `_registryServer.mjs`
+
+    await xfs.mktempPromise(async cwd => {
       await expect(runCli(cwd, [`yarn@stable`, `--version`], true)).resolves.toMatchObject({
         exitCode: 1,
-        stderr: /Signature does not match/,
+        stderr: expect.stringMatching(/Signature does not match/),
```

Comment:
nit
```suggestion
        stderr: expect.stringContaining(`Signature does not match`),
```

Reply:
I’ve changed it to this style everywhere.